pkg/proxy/ipvs/proxier.go

```diff
@@ -80,11 +76,14 @@ const (
 
 // tableChainsWithJumpService is the iptables chains ipvs proxy mode used.
 var tableChainsWithJumpService = []struct {
```

Comment:
This name doesn't seem tru any more - it's just a table of jumps.

Needs a better comment, too.